Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN npm run build
 ########
 # Python dependencies builder
 #
-FROM python:3.7-slim-buster AS python-builder
+FROM python:3.9-slim-buster AS python-builder
```

Comment:
Should we also upgrade to `python:3.9-slim-bullseye`?

Reply:
DMed about it and going to try bullseye

- [ ] @stevejalim switch to `bullseye`